### PR TITLE
feat: Move survey to the footer 

### LINKED ISSUE #544: Move survey to the footer
The top survey bar should be removed from the top navigation bar and should be added in the footer as a link.

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #544 

## Demo
image
image

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] Move survey to the footer

### CODE REVIEW

**Review by Liubov-crypto — APPROVED** [review]
LGTM

File: src/components/TheFooter.vue [hunk 1]
```diff
@@ -105,12 +105,14 @@ const links = {
     { label: 'Blog', url: 'https://blog.aeternity.com' },
     { label: 'Terms of Service', url: '/terms-of-service' },
     { label: 'Privacy Policy', url: '/privacy-policy' },
+    { label: 'Fill out the Survey', url: 'https://aeternity.com/aescan-feedback-survey' },
```
Comment (michele-franchi) [inline src/components/TheFooter.vue:108]:
```suggestion
    { label: 'Feedback Survey', url: 'https://aeternity.com/aescan-feedback-survey' },
```

I think from a first look it makes more clear what the survey is about.

**Review by michele-franchi — APPROVED** [review]
I approve in advance but I would adjust the footer text.

File: src/components/TheFooter.vue (hunk 1 above)
Comment (janmichek) [reply src/components/TheFooter.vue]: Yes agree. Fixed

**Pull request merged**